docs/src/content/docs/zh/principles/sftp.md

```
@@ -3,15 +3,15 @@ title: SFTP 系统
 description: SFTP 文件浏览器的工作原理
 ---
 
-SFTP 系统通过 SSH 提供文件管理功能。
+SFTP 通过 SSH 管理远程文件。
 
 ## 架构
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)